apis/Google.Devtools.AspNet/Google.Devtools.AspNet/CloudTrace/BufferingTraceConsumer.cs

```diff
@@ -23,7 +23,7 @@ namespace Google.Devtools.AspNet
     /// </summary>
     internal sealed class BufferingTraceConsumer : IFlushableTraceConsumer
     {
-        // The default buffer size in bytes. 2^16 = 65536.
+        /// <summary>The default buffer size in bytes. 2^16 = 65536.</summary>
```

Comment:
If you do want to use this pattern, there are many other fields in this PR which should have it as well.